tracker/views.py

```diff
@@ -43,3 +45,52 @@ def post(self, request, *args, **kwargs) -> HttpResponse:
         [messages.error(request, error_) for error_ in form.errors.values()]
 
         return render(request, self.template_name, {"form": form})
+
+
+class ContributorListView(ListView):
```

Comment:
We will manage all contributors via admin site so please move your logic in `tracker/admin.py`

Reply:
addressed in [ea8a970](https://github.com/KostCode-Organization/hackathon_bot/pull/44/commits/ea8a970dbdf23b0c8fabf29e52fdd189d3ff6fe6)